src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -25,17 +31,15 @@
     <IncludeSymbols>true</IncludeSymbols>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <IsWindows Condition="$([MSBuild]::IsOSPlatform('Windows'))">true</IsWindows>
-    <IsOSX Condition="$([MSBuild]::IsOSPlatform('OSX'))">true</IsOSX>
-    <IsLinux Condition="$([MSBuild]::IsOSPlatform('Linux'))">true</IsLinux>
-  </PropertyGroup>
-
   <Target Name="PrintOS" BeforeTargets="Build">
     <Message Text="IsWindows: '$(IsWindows)'" Importance="high" />
     <Message Text="IsOSX: '$(IsOSX)'" Importance="high" />
     <Message Text="IsLinux: '$(IsLinux)'" Importance="high" />
   </Target>
```

Comment:
This seems like debug info from troubleshooting the build. Is there a reason to still muddy the build output by dumping out these variables?

Reply:
This is information logs, Robert asked a similar question here: https://mongodb.slack.com/archives/GQ27C9N7J/p1606854006080700. In two words, I think we can remove it, but at the same time, I think this still be helpful from time to time. Since It's harmless, I would not do it, but I don't insist on it

Reply:
Removed, was a remnant of some debugging.